src/main/java/de/zalando/aruha/nakadi/controller/EventPublishingController.java

```diff
@@ -55,4 +74,15 @@ public ResponseEntity postEvent(@PathVariable final String eventTypeName, @Reque
         }
     }
 
+    private Optional<ValidationError> validateSchema(final String event, final EventType eventType) {
+        try {
+            final ValidationStrategy validationStrategy = new EventBodyMustRespectSchema();
+            final ValidationStrategyConfiguration vsc = new ValidationStrategyConfiguration();
```

Comment:
This class plays no role in the validation, but I had to create it anyway since it's required.

I tried not to mess up with the existing validations code since it covers lots of future needs.